js/mainViewModel.js

```diff
@@ -89,6 +78,7 @@ module.exports = function MainViewModel(butler, linkifyEmitter, routingEmitter)
 			}
 		})
 		butler.refreshPost(key)
+		butler.refreshPost(config.template)
```

Comment:
hmm, I don't think this needs to be refreshed on every navigation.  The butler will already refresh each post periodically, including these templates.